frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelDocs.tsx

```diff
@@ -1,13 +1,93 @@
+import { themeLogic } from '../../themeLogic'
+
+function SidePanelDocsSkeleton(): JSX.Element {
+    return (
+        <div className="absolute inset-0 p-4 space-y-2">
+            <LemonSkeleton className="w-full h-10 mb-12" />
+            <LemonSkeleton className="w-1/3 h-8" />
+            <LemonSkeleton className="w-1/2 h-4 mb-10" />
+            <LemonSkeleton className="w-full h-4" />
+            <LemonSkeleton className="w-full h-4 opacity-80" />
+            <LemonSkeleton className="w-full h-4 opacity-60" />
+            <LemonSkeleton className="w-full h-4 opacity-40" />
+            <LemonSkeleton className="w-1/2 h-4 opacity-20" />
+        </div>
+    )
+}
 
 export const SidePanelDocs = (): JSX.Element => {
-    const { path } = useValues(sidePanelDocsLogic)
+    const { iframeSrc, currentUrl } = useValues(sidePanelDocsLogic)
+    const { updatePath, unmountIframe, closeSidePanel, handleExternalUrl } = useActions(sidePanelDocsLogic)
+    const ref = useRef<HTMLIFrameElement>(null)
+    const [ready, setReady] = useState(false)
+    const { isDarkModeOn } = useValues(themeLogic)
+
+    useEffect(() => {
+        ref.current?.contentWindow?.postMessage(
+            {
+                type: 'theme-toggle',
+                isDarkModeOn,
+            },
+            '*'
+        )
+    }, [isDarkModeOn, ref.current])
+
+    useEffect(() => {
+        const onMessage = (event: MessageEvent): void => {
+            if (event.origin === POSTHOG_WEBSITE_ORIGIN) {
+                if (event.data.type === 'internal-navigation') {
+                    updatePath(event.data.url)
+                    return
+                }
+                if (event.data.type === 'docs-ready') {
+                    setReady(true)
+                    return
+                }
+
+                if (event.data.type === 'external-navigation') {
+                    // This should only be triggered for app|eu.posthog.com links
+                    handleExternalUrl(event.data.url)
+                    return
+                }
+
+                console.warn('Unhandled iframe message from Docs:', event.data)
+            }
+        }
+
+        window.addEventListener('message', onMessage)
+
+        return () => window.removeEventListener('message', onMessage)
+    }, [ref.current])
+
+    useEffect(() => {
+        return unmountIframe
+    }, [])
 
-    // NOTE: Currently we can't detect url changes from the iframe
     return (
-        <div className="w-full h-full overflow-hidden">
-            <iframe src={`https://posthog.com${path ?? ''}`} title="Docs" className="w-full h-full" />
-        </div>
+        <>
+            <SidePanelPaneHeader>
+                <LemonButton
+                    size="small"
+                    sideIcon={<IconExternal />}
+                    to={currentUrl}
+                    targetBlank
+                    onClick={() => closeSidePanel()}
+                >
+                    Open on posthog.com
```

Comment:
This _already_ is posthog.com. My feeling is that plain "Open in new tab" would be significantly clearer

Reply:
Sure